lit/Swift/Refcount.swift

```diff
@@ -0,0 +1,31 @@
+// REQUIRES: darwin
```

Comment:
I found that some older lit tests did not really require Darwin (see: https://github.com/apple/swift-lldb/pull/401). Are we sure we need that here?

Reply:
Not quite needed actually, just muscle memory and fat fingering. Let me fix it.